src/components/Profile/index.js

```diff
@@ -90,7 +93,16 @@ const Profile = props => {
         </div>
         <InterestsGrid interestList={userInterests} />
       </div>
-
+      <div className="start-conversation-button">
+        {!ownProfile && startDirectChannel && (
+          <ButtonContainer
```

Comment:
I would probably put some more margin between the button and interests, but that's just personal preference

Reply:
Good point, added some more